avm/res/azure-stack-hci/cluster/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -0,0 +1,98 @@
+targetScope = 'subscription'
+
+metadata name = 'Deploy Azure Stack HCI Cluster in Azure with a 2 node switched configuration WAF aligned'
+metadata description = 'This test deploys an Azure VM to host a 2 node switched Azure Stack HCI cluster, validates the cluster configuration, and then deploys the cluster WAF aligned.'
+
+@description('Optional. Location for all resources.')
+param location string = deployment().location
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-azure-stack-hci.cluster-${serviceShort}-rg'
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'ashc2nwaf'
+
+@description('Optional. A token to inject into the name of each resource.')
+param namePrefix string = '#_namePrefix_#'
+
+@description('Optional. The password of the LCM deployment user and local administrator accounts.')
+@secure()
+param localAdminAndDeploymentUserPass string = newGuid()
+
+@description('Required. The app ID of the service principal used for the Azure Stack HCI Resource Bridge deployment. If omitted, the deploying user must have permissions to create service principals and role assignments in Entra ID.')
+@secure()
+#disable-next-line secure-parameter-default
+param arbDeploymentAppId string = ''
```

Comment:
Just a blocking comment for us maintainers until the same secrets are set up

Reply:
@AlexanderSehr  The HCI RP identity doesn't show up until you register for the Microsoft.AzureStackHCI RP.  Once registered, running this command (or the CLI equivalent) should show the SP ID for the global app in the current tenant:  Get-AzADServicePrincipal -ApplicationId 1412d89f-b8a8-4111-b4fd-e82905cbd85d

I'll add the app ID to the param description...

Reply:
Hey @mbrat2005,
I'm really sorry this is taking so long on my end. I'm trying to squeeze working on this PR in every time I get the chance. I'll try to continue on this this week 💪